app/models/reason.rb

```diff
@@ -0,0 +1,3 @@
+class Reason < ActiveRecord::Base
```

Comment:
I'm guessing your first sentence here was going to be:

    I seriously think we should NOT have this model. 

If that's the case... I don't think that having to write a query to fetch Reasons for EvidenceChecks having failed:

- because this app will not have that many users that millisecond optimisations are going to hold us back
- it never hurts to have the data organised in separate components
- in the future the reason text field will potentially be replaced with a radio box, by having it in a separate table it's easier to extend/add that functionality 
- as for the CamelCasedName of **EvidenceCheckInvalidReason** I think that it looks ugly. The 'belongs_to' clearly describes what the table relates to and I find this easier:

```ruby
evidence.reason.explanation
```
than
```ruby
evidence.evidencecheckinvalidreason.explanation
```